### PR TITLE
DEVDOCS-5996: [Update] channel ID is no longer a requirement for toke…

### PR DESCRIPTION
**Pull request by bc-traciporter**
…n creation

# [DEVDOCS-5996]

## What changed?

* removed text regarding the requirement to enter a channel ID when creating a Storefront token
* Fixed incorrect example
* Created a ticket to remove duplicated request fields.

## Release notes draft

We're happy to announce that you can use a channel ID or channel IDs  when creating a Storefront token. (needs review)

* 

## Anything else?

ping {names}

[DEVDOCS-5996]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5996?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/storefront_tokens.v3.yml [hunk 1]
```diff
@@ -179,7 +184,6 @@ components:
           example: 1885635176
           minimum: 0
       required:
-        - channel_id
```
Comment (bc-rafsierk) [inline reference/storefront_tokens.v3.yml:182]: Right now it is a little bit tricky, as we expect to receive either channel_id or channel_ids. Not sure how to write it

Comment (bc-traciporter) [reply reference/storefront_tokens.v3.yml:182]: I updated the schema to accommodate this option.

File: reference/storefront_tokens.v3.yml [hunk 2]
```diff
@@ -169,18 +174,22 @@ components:
...
+          description: Channel ID or list of channel IDs for requested token
         expires_at:
           type: integer
           description: Unix timestamp (UTC time) defining when the token should expire. Supports seconds, but does not support milliseconds, microseconds, or nanoseconds.
           example: 1885635176
           minimum: 0
       required:
-        - channel_id
         - expires_at
+        - channel_id
```
Comment (bc-rafsierk) [inline reference/storefront_tokens.v3.yml:192]: ok I was not clear enough 
we require either `channel_ids` or `channel_id` . `channel_ids` is of array type, `channel_id` is integer type. 

Let mention only channel_ids then

**Review by bc-rafsierk — DISMISSED** [review]

File: reference/storefront_tokens.v3.yml [hunk 3]
```diff
@@ -169,18 +174,23 @@ components:
...
-          description: Channel ID for requested token
-          example: 1
+           type: integer
+           example: 1
+           description: Channel ID for a requested token. Use this field to enter a channel ID. Do not use this field if you have more than one channel. You can not use both `channel_id` and `channel_ids` in your request.
+        channel_ids:
+           type: array
+           items:
+             type: integer
+             example: [667251, 1]
```
[the remark this answers is not in the document]
Comment (bc-rafsierk) [reply reference/storefront_tokens.v3.yml:184]: only this one to fix and :+1:

**Review by bc-tgomez — APPROVED** [review]

**Pull request merged**